tests/test_hooks/test_checkpoint_hook.py

```diff
@@ -539,8 +576,46 @@ def test_with_runner(self, training_type):
                              f'best_test_acc_{training_type}_5.pth')
         self.assertTrue(osp.isfile(best_ckpt))
 
+        ckpt = torch.load(osp.join(cfg.work_dir, f'{training_type}_5.pth'))
+        self.assertEqual(best_ckpt,
+                         ckpt['message_hub']['runtime_info']['best_ckpt'])
+
+        if training_type == 'epoch':
+            self.assertEqual(ckpt['meta']['epoch'], 5)
+            self.assertEqual(ckpt['meta']['iter'], 20)
+        else:
+            self.assertEqual(ckpt['meta']['epoch'], 0)
+            self.assertEqual(ckpt['meta']['iter'], 5)
```

Comment:
Is it necessary to check these intermediate variable here?

Reply:
Those variables may be incorrect if they are not handled carefully.